extras/fixture/src/unity_fixture.c

```diff
@@ -7,7 +7,9 @@
 
 #include "unity_fixture.h"
 #include "unity_internals.h"
+#include <stdio.h>
```

Comment:
Yup I realised I can use UnityPrintNumberUnsigned, so I can now remove this #include.